packages/nc-gui-v2/components/dashboard/GithubStarButton.vue

```diff
@@ -0,0 +1,9 @@
+<script setup lang="ts">
+import GithubButton from 'vue-github-button'
+</script>
+
+<template>
+  <GithubButton href="https://github.com/nocodb/nocodb" data-icon="octicon-star" :data-show-count="true" data-size="large"
+    >Star</GithubButton
+  >
```

Comment:
Broken formatting

It should look like this instead
```vue
<GithubButton href="https://github.com/nocodb/nocodb" data-icon="octicon-star" :data-show-count="true" data-size="large">
	Star
</GithubButton>
```

Reply:
Caused by `npm run lint`. Ignore it first.

Reply:
Yeah eslint thinks the "broken" formatting (as I like to call it) is okay in some cases, when the indentation isn't 

```
<tag>
	content
</tag>
```

but instead something like
```
<tag>a lot of content here foobarbazbazfoo</tag>
```

Which I personally find a bit frustrating because it's not that readable when it looks like
```
<tag
	> content </tag
>
```